### PR TITLE
Adds the Privacy Manifest to the list of processed files

### PR DESCRIPTION
**Pull request by dbarden**
## Purpose

Adds the Privacy Manifest file to Package.swift

## Checklist
- [ ] The change was thoroughly tested manually
- [ ] The change was covered with unit tests
- [ ] The change was tested with the integration example
- [ ] Necessary changes were made in the integration example (if applicable)
- [ ] New functionality is reflected in README (if applicable)

### CODE REVIEW

File: Package.swift [hunk 1]
```diff
@@ -1,4 +1,4 @@
-// swift-tools-version:5.1
+// swift-tools-version:5.3
```
Comment (dbarden) [inline Package.swift:1]: This was the minimum version to allow adding the `resources` field

**Review by vijaytholpadi — APPROVED** [review]
Tried out this solution and it fixes the issue of missing Privacy Manifest incase of SPM integration.

**Comment by vijaytholpadi** [comment]
@viaskal-sift Can we get this PR merge and a release scheduled pls? 

The Privacy manifest for the Sift library is missing in our app's migration from CocoaPods to SPM due to this issue.

**Comment by viaskal-sift** [comment]
@vijaytholpadi yes, we will release it probably next week. Would it be ok for you?

**Comment by vijaytholpadi** [comment]
@viaskal-sift Checking in as it has been a week, can a release be made please?

**Comment by viaskal-sift** [comment]
@vijaytholpadi This one will be released either today, or tomorrow

**Review by iprysiazhnyi-sift — APPROVED** [review]

**Comment by iprysiazhnyi-sift** [comment]
a new released (v2.2.1) published with this change.
Thanks everyone!

**Pull request merged**